remediation/self-node-remediation/known-issues.md

```diff
@@ -10,10 +10,8 @@ nav_order: 4
 Self Node Remediation has several known issues:
 1. Currently only one health detection system (e.g. NHC, MHC) is supported at the same time (i.e. you can't use NHC and MHC at the same time)
 2. The timeout to assume the node has been rebooted is the same for all nodes. The safe timeout should be configured for the highest watchdog timeout
-3. Self Node Remediation does not run on masters
-4. Uninstalling the operator doesn't remove the self node remediation daemonset. a user should delete the SelfNodeRemediationConfig CR to remove the daemonset
-5. Upon self node remediation installation, it might take up to 2 minutes before the daemonset is deployed
-6. IPv6 is not supported
-7. Clusters with OVN network plugin are not supported due to [Bug 2068910](https://bugzilla.redhat.com/show_bug.cgi?id=2068910)
-8. At least two workers are required to get node fencing
-9. At release 0.3 channel name was changed from "**alpha**" to "**stable**". This means that when releases prior to 0.3 are upgraded, the channel name needs to be changed manually 
+3. Uninstalling the operator doesn't remove the self node remediation daemonset. a user should delete the SelfNodeRemediationConfig CR to remove the daemonset
+4. Upon self node remediation installation, it might take up to 2 minutes before the daemonset is deployed
+5. IPv6 is not supported
+6. At least two workers are required to get node fencing
```

Comment:
makes sense, but do we actually prevent remediating a single worker node? 🤔

Reply:
It's not enforced by SNR .
This paragraph is pretty old, so I assume it's either more of a guideline to the user or enforced by NHC ? :thinking:

Reply:
Also I'm not entirely sure it would be true.
ATM I think we can technically fence / remediate a single worker node.
It'll probably get the cluster pretty jammed up but maybe some users are ok with the risk / trade-off :man_shrugging: